src/shared/DotnetTool/dotnet-tool.nuspec

```diff
@@ -5,11 +5,14 @@
         <version>$version$</version>
         <description>Secure, cross-platform Git credential storage with authentication to Azure Repos, GitHub, and other popular Git hosting services.</description>
         <authors>git-credential-manager</authors>
+        <icon>images\icon.png</icon>
+        <iconUrl>https://raw.githubusercontent.com/GitCredentialManager/git-credential-manager/main/assets/gcm-transparent.png</iconUrl>
```

Comment:
`iconUrl` is deprecated, but `icon` is only supported by newer versions of NuGet and VS.. so include both, with `iconUrl` as a fallback.

Newer tools will only use `icon`.